code/modules/client/client procs.dm

```diff
@@ -362,14 +362,17 @@
 
 /client/proc/log_client_to_db()
 
-	if( IsGuestKey(src.key) )
+	if( IsGuestKey(key) )
 		return
 
+	// Isn't the ckey sanitized by default?
+	var/sql_ckey = sanitizeSQL(ckey)
```

Comment:
I added it because I don't trust byond. An auth exploit could allow somebody to forge a ckey with invalid data in it.